### ISSUE TITLE
Cannot supply multiple files to pact-verifier

### ISSUE THREAD

**Issue by kevinsimons-wf** [pact-foundation/pact-python#21]
According to the [README](https://github.com/pact-foundation/pact-python#--pact-urls), to supply multiple local pact files you give the `--pact-urls` argument a comma separated list of files.

However, this will give you an error that the file cannot be found and that file is the combined paths, 
```
Error: The following Pact files could not be found:
/tmp/pacts/pact1.json,/tmp/pacts/pact2.json
```
when they should be separated by a [new line](https://github.com/pact-foundation/pact-python/blob/master/pact/verify.py#L69).

This is because for the click option `--pact-urls` the parameter `multiple=True` is used. However this is meant for providing multiple `--pact-urls` flags such as
```
pact-verifier --pact-urls=./provider/pact1.json --pact-urls=./provider/pact2.json
```
http://click.pocoo.org/5/options/#multiple-options

Either the documentation should be updated to list that you should provide an argument for each file, or remove `multiple=True` and just do a `.split(',')` when reading the pact-urls argument.

**Comment by bethesque** [comment]
I much prefer `- -pact-url path - - pact-url otherpath`. If the underlying ruby command line tool would support that, it's what I'd have used in the ruby interface rather than comma delimited, which is ugly with URLs.

**Comment by bethesque** [comment]
Sorry, ignore spaces between dashes, on phone.

**Comment by matthewbalvanz-wf** [comment]
Resolved in v0.7.0
